cross/build-rootfs.sh

```diff
@@ -20,27 +26,16 @@ __UbuntuPackages+=" symlinks"
 
 # CoreCLR and CoreFX dependencies
 __UbuntuPackages+=" gettext"
-__UbuntuPackages+=" libcurl4-openssl-dev"
+__UbuntuPackages+=" libunwind8-dev"
+__UbuntuPackages+=" liblttng-ust-dev"
 __UbuntuPackages+=" libicu-dev"
+
+# CoreFX dependencies
+__UbuntuPackages+=" libcurl4-openssl-dev"
 __UbuntuPackages+=" libkrb5-dev"
-__UbuntuPackages+=" liblttng-ust-dev"
 __UbuntuPackages+=" libssl-dev"
-__UbuntuPackages+=" libunwind8-dev"
 __UbuntuPackages+=" zlib1g-dev"
 
-if [ -z "$LLVM_ARM_HOME" ]; then
-    __LLDB_Package="lldb-3.6-dev"
-fi
- 
-
-
-__BuildArch=arm
-__UbuntuArch=armhf
-__LinuxCodeName=trusty
-__UbuntuRepo="http://ports.ubuntu.com/"
-__MachineTriple=arm-linux-gnueabihf
```

Comment:
@mellinoe `__MachineTriple` is removed, since there is no use. Other three variables are moved to the beginning of script to impove readibliity of the script and to maintain consistency with other build-rootfs.sh in coreclr and core-setup.